src/apps/statements/tests/storingStatements/queriables/getMetadataFromStatement/getSequencingMetadata.test.ts

```diff
@@ -0,0 +1,61 @@
+import * as assert from 'assert';
+import { assign } from 'lodash';
+
+import { getSequencingMetadata } from '../../../../service/storeStatements/queriables/getMetadataFromStatement/getSequencingMetadata';
+import { interactionActivityStatement, statementDefaults } from './fixtures/statements.fixture';
+
+describe(
+  'Retrieve duration metadata from statement',
+  () => {
+    it(
+      'should return choices with proper order(sequence)',
+      () => {
+        assert
+          .equal(
+            getSequencingMetadata(
+              assign(
+                {},
+                interactionActivityStatement,
+                {
+                  result: {},
+                },
+              ),
+            ),
+            false,
+          );
+
+        assert
+          .equal(getSequencingMetadata(statementDefaults), false);
+
+        assert
+          .deepEqual(
+            getSequencingMetadata(
+              assign(
+                {},
+                interactionActivityStatement,
+                {
+                  result: {
+                    response: 'tim',
+                  },
+                },
+              ),
```

Comment:
I think this is the last one where we should be able to use the spread operator.